libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/distanceinfo/LineDistanceInfo.kt

```diff
@@ -0,0 +1,76 @@
+package com.mapbox.navigation.base.trip.model.roadobject.distanceinfo
+
+import com.mapbox.navigation.base.trip.model.roadobject.RoadObjectType
+
+/**
+ * LineDistanceInfo contains information about distance to the road object represented as line.
+ *
+ * @param distanceToEntry distance to the entry of the object
+ * @param distanceToExit distance to the exit of the object
+ * @param distanceToEnd distance to the end of the object
+ * @param entryFromStart true if we enter object from the start
```

Comment:
What does this mean? When this is false, how can we enter something from the exit? Let's explain in the docs.

Reply:
cc @etl @SiarheiFedartsou

Reply:
As I see [in code](https://github.com/mapbox/mapbox-navigation-native/blob/master/src/eh/manager.cpp#L320), yes. Guess, it might happen if to add a line and go into it from another side